tests/foreman/cli/test_operatingsystem.py

```diff
@@ -358,31 +356,34 @@ def test_positive_update_parameters_attributes(self):
         assert param_name == os['parameters'][0]['name']
         assert param_value == os['parameters'][0]['value']
 
-    @pytest.mark.destructive
-    @pytest.mark.skip_if_open("BZ:1649011")
-    def test_positive_os_list_with_default_organization_set(self):
-        """list operating systems when the default organization is set
 
-        :id: 2c1ba416-a5d5-4031-b154-54794569a85b
+@pytest.mark.tier2
+@pytest.mark.skip_if_open("BZ:1649011")
+def test_positive_os_list_with_default_organization_set(satellite_latest):
```

Comment:
Loving this pattern of explicitly interacting with the satellite instance resolved for this test function!